### ISSUE TITLE
Using CategoryPublisher for update-site creation

### ISSUE THREAD

**Issue by m273d15** [diffplug/goomph#124]
Hi, thanks for your great plugin.

We use goomph to resolve the osgi dependencies and to build
the plugin's update-site. However, I had to write a new wrapper `CategoryPublisher`
(similar to your `FeaturesAndBundlesPublisher`; see below).

Therefore, my question: Do you plan to support the CategoryPublisher feature in the future (or is it already supported and I missed it)?
(I assume also other users would like to create an update-site) 

-------------
```java
import com.diffplug.gradle.FileMisc;
import com.diffplug.gradle.eclipserunner.EclipseApp;
import com.diffplug.gradle.pde.EclipseRelease;
import com.diffplug.gradle.pde.PdeInstallation;
import java.io.File;

public class CategoryPublisher extends EclipseApp {

  private final String eclipseVersion;

  public CategoryPublisher(String eclipseVersion) {
    super("org.eclipse.equinox.p2.publisher.CategoryPublisher");
    this.eclipseVersion = eclipseVersion;
  }

  public void metadataRepository(File file) {
    addArg("metadataRepository", FileMisc.asUrl(file));
  }

  public void categoryDefinition(File file) {
    addArg("categoryDefinition", FileMisc.asUrl(file));
  }

  public void runUsingPdeInstallation() throws Exception {
    runUsing(PdeInstallation.from(EclipseRelease.official(eclipseVersion)));
  }
}

```

**Comment by nedtwigg** [comment]
> Do you plan to support the CategoryPublisher feature in the future (or is it already supported and I missed it)?

It is not already supported, and we at DiffPlug do not have plans to add it.  We would be happy to merge a PR from you however, and we'll make sure that any accompanying unit tests continue to pass.

**Comment by m273d15** [comment]
Thank you, I will check how I could test my class with your test harness and open a PR in the next days.

**Comment by nedtwigg** [comment]
Published in `3.24.0`
